css-brochure-form/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2>LEARN MORE</h2>
+    <div></div>
+    <form>
+      <div>
+        <label style="margin-left: 10px" for="email">COMPANY EMAIL</label>
+        <br />
+        <input
+          class="input"
+          type="email"
+          id="email"
+          name="name"
+          placeholder="sally@company.co" />
+      </div>
+      <div>
+        <label style="margin-left: 10px" for="name">NAME</label>
+        <br />
+        <input
+          class="input"
+          type="text"
+          id="name"
+          name="name"
+          placeholder="Sally Smith" />
+      </div>
+      <div>
+        <label style="margin-left: 10px" for="hiring"
+          >WHERE ARE YOU HIRING?</label
+        >
+        <br />
+        <input
+          class="input"
+          type="text"
+          id="location"
+          name="location"
+          placeholder="City, Country" />
+      </div>
+      <div>
+        <label style="margin-left: 10px" for="engineers-select"
+          >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+        >
+        <br />
+        <select
+          style="background-color: #e7e7e7"
+          class="input"
+          name="engineers"
+          id="engineers-select">
+          <option value="1-2">1-2 Engineers</option>
+          <option value="3-4">3-4 Engineers</option>
+          <option value="5-6">5-6 Engineers</option>
+          <option value="7-10">7-10 Engineers</option>
+        </select>
+      </div>
+      <span style="float: right"><button>SUBMIT</button></span>
```

Comment:
You want to avoid `float` as much as possible. It's bad practice. You should change the `span` to a `div` and give the `div` `text-align: right`